.circleci/real_config.yml

```
@@ -577,7 +577,7 @@ commands:
             # of the tests that would be run, along with some other stuff. Each file containing
             # any selected tests is printed on a line by itself, except for (for some reason) a
             # single trailing space.
-            pytest --setup-plan -m '<<parameters.mark>>' | egrep '\.py (ss)?$' | sed 's/ \(ss\)\{0,1\}$//' > /tmp/all-relevant-files
+            pytest --setup-plan -m '<<parameters.mark>>' | egrep '\.py s*$' | sed 's/\.py.*$/.py/' > /tmp/all-relevant-files
```

Technically, the grep isn't necessary here. One can tell sed to only print lines where the substitution happens. :)

Either way is fine; just less repetition this way.
```suggestion
            pytest --setup-plan -m '<<parameters.mark>>' | sed -n 's/(\.py) s*$/\1/p' > /tmp/all-relevant-files
```

---

interesting, I do think that is a cleaner solution

though I get
```
sed: -e expression #1, char 17: invalid reference \1 on `s' command's RHS
```

my quick attempts to fix this didn't work either, so just in the interest of spending time wisely I am going to keep the working solution as is